src/ProxyClient/Varnish.php

```diff
@@ -150,7 +158,10 @@ protected function configureOptions()
         $resolver = parent::configureOptions();
         $resolver->setDefaults([
             'tags_header' => self::DEFAULT_HTTP_HEADER_CACHE_TAGS,
+            'tags_purge_header' => self::DEFAULT_HTTP_PURGE_HEADER_CACHE_TAGS,
             'header_length' => 7500,
+            'tags_invalidation_by_purge' => false, // false, true or "single" for purge invalidation per tag
+            'tags_separator' => ',',
```

Comment:
Can we get rid of this option? For ban, our library can just use commas. It seems the combined surrogate keys will be [space-delimited](https://github.com/varnish/varnish-modules/pull/47), so no choice there either.